esp-hal/src/dma/mod.rs

```diff
@@ -556,6 +556,7 @@ macro_rules! as_mut_byte_array {
         unsafe { &mut *($name.as_mut_ptr() as *mut [u8; $size]) }
     };
 }
+pub use as_mut_byte_array; // TODO: can be removed as soon as DMA is stabilized
```

Comment:
The weird `pub use` on macros is a solution to the following error:

```
error: macro-expanded `macro_export` macros from the current crate cannot be referred to by absolute paths
   --> C:\_Espressif\esp-hal\esp-hal\src\spi\master.rs:969:21
    |
969 |                     crate::as_mut_byte_array!(BUFFERS[id], 4)
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #52234 <https://github.com/rust-lang/rust/issues/52234>
note: the macro is defined here
   --> C:\_Espressif\esp-hal\esp-hal\src\dma\mod.rs:554:1
    |
554 | / macro_rules! as_mut_byte_array {
555 | |     ($name:expr, $size:expr) => {
556 | |         unsafe { &mut *($name.as_mut_ptr() as *mut [u8; $size]) }
557 | |     };
558 | | }
    | |_^
    = note: `#[deny(macro_expanded_macro_exports_accessed_by_absolute_paths)]` on by default
```